src/app/view/components/dot-dialog-message/services/dot-dialog-message.service.ts

```diff
@@ -0,0 +1,40 @@
+import { BehaviorSubject, Observable } from 'rxjs';
+import { Injectable } from '@angular/core';
+import { DotcmsEventsService, DotEventData } from 'dotcms-js';
+
+export interface DotDialogMessageParams {
+    title: string;
+    width?: string;
+    height?: string;
+    body?: string;
+    code?: {
+        lang: string;
+        content: string;
+    };
+}
+
+@Injectable()
+export class DotDialogMessageService {
+    private _messages: BehaviorSubject<DotDialogMessageParams> = new BehaviorSubject(null);
+
+    constructor(dotcmsEventsService: DotcmsEventsService) {
+        dotcmsEventsService.subscribeTo('LARGE_MESSAGE').subscribe((messageEvent: DotEventData) => {
+            const { code, width, body, title, lang, height } = messageEvent.data;
+            this.push({
+                title,
+                height,
+                width,
+                body,
+                code: { lang, content: code }
+            });
+        });
+    }
+
+    push(message: DotDialogMessageParams): void {
```

Comment:
We don't need this. We just need to expose whatever this `dotcmsEventsService.subscribeTo('LARGE_MESSAGE')` returns.